include/deal.II/matrix_free/vector_access_internal.h

```diff
@@ -0,0 +1,683 @@
+// ---------------------------------------------------------------------
```

Comment:
I guess you did not want to pull in this file from our matrix-free development branch?

Reply:
This was a side effect of cherry-pick (and tiredness)... Sorry about this!